tests/metrics_api_SUITE.erl

```diff
@@ -67,108 +73,158 @@ message_flow(Config) ->
-                assert_counter_inc(1, PresRecv1, PresRecv2),
-                StanzaSent2 = fetch_counter_value(xmppStanzaSent, Config),
-                assert_counter_inc(1, StanzaSent1, StanzaSent2),
-                StanzaRecv2 = fetch_counter_value(xmppStanzaReceived, Config),
-                assert_counter_inc(1, StanzaRecv1, StanzaRecv2)
-        end).
-
-presence_direct_one(Config) ->
-    escalus:story(Config, [{alice, 1}, {bob, 1}], fun(Alice, Bob) ->
-                PresSent1 = fetch_counter_value(xmppPresenceSent, Config),
-                PresRecv1 = fetch_counter_value(xmppPresenceReceived, Config),
-                StanzaSent1 = fetch_counter_value(xmppStanzaSent, Config),
-                StanzaRecv1 = fetch_counter_value(xmppStanzaReceived, Config),
-
-                Presence = escalus_stanza:presence_direct(bob, <<"available">>),
-                escalus:send(Alice, Presence),
-                escalus:wait_for_stanza(Bob),
-
-                PresSent2 = fetch_counter_value(xmppPresenceSent, Config),
-                assert_counter_inc(1, PresSent1, PresSent2),
-                PresRecv2 = fetch_counter_value(xmppPresenceReceived, Config),
-                assert_counter_inc(1, PresRecv1, PresRecv2),
-                StanzaSent2 = fetch_counter_value(xmppStanzaSent, Config),
-                assert_counter_inc(1, StanzaSent1, StanzaSent2),
-                StanzaRecv2 = fetch_counter_value(xmppStanzaReceived, Config),
-                assert_counter_inc(1, StanzaRecv1, StanzaRecv2)
-        end).
-
-iq_one(Config) ->
-    escalus:story(Config, [{alice, 1}], fun(Alice) ->
-                IqSent1 = fetch_counter_value(xmppIqSent, Config),
-                IqRecv1 = fetch_counter_value(xmppIqReceived, Config),
-                StanzaSent1 = fetch_counter_value(xmppStanzaSent, Config),
-                StanzaRecv1 = fetch_counter_value(xmppStanzaReceived, Config),
-
-                RosterIq = escalus_stanza:roster_get(),
-                escalus_client:send(Alice, RosterIq),
-                escalus_client:wait_for_stanza(Alice),
-
-                IqSent2 = fetch_counter_value(xmppIqSent, Config),
-                assert_counter_inc(1, IqSent1, IqSent2),
-                IqRecv2 = fetch_counter_value(xmppIqReceived, Config),
-                assert_counter_inc(1, IqRecv1, IqRecv2),
-                StanzaSent2 = fetch_counter_value(xmppStanzaSent, Config),
-                assert_counter_inc(1, StanzaSent1, StanzaSent2),
-                StanzaRecv2 = fetch_counter_value(xmppStanzaReceived, Config),
-                assert_counter_inc(1, StanzaRecv1, StanzaRecv2)
-        end).
-
-sessions(Config) ->
-    escalus:story(Config, [{alice, 2}, {bob, 1}], fun(_Alice1, _Alice2, _Bob) ->
-        3 = fetch_global_counter_value(totalSessionCount, Config),
-        2 = fetch_global_counter_value(uniqueSessionCount, Config)
-    end).
+
+one_client_just_logs_in(Config) ->
+    instrumented_story
```

Comment:
I would do it like:

```erlang
S = fun (JarekKrolWiertarek) ->
        ...
    end.
instrumented_story(..., S).
```
Breaking line just after the opening parentheses looks ugly (my personal opinion).